### PR TITLE
show user flags in timeline

### PR DESCRIPTION
**Pull request by incognitojam**
Add styles for user flags in timeline

The UI already makes it easy to support new event types, so once the backend events.json includes user flags they will appear in the timeline without much effort
Screenshot from 2022-08-22 17-33-07

### CODE REVIEW

**Comment by incognitojam** [comment]
Screenshot from 2022-08-23 15-38-07

**Comment by incognitojam** [comment]
Updated to yellow and showing with various route lengths
Screenshot from 2022-08-24 10-35-49

File: src/actions/cached.js [hunk 1]
```diff
@@ -202,6 +207,53 @@ function parseEvents(route, driveEvents) {
...
+        prev.data.end_route_offset_millis = ev.route_offset_millis;
+        res.push(prev);
+
+        currOverride = {
+          ...currOverride,
+          route_offset_millis: currFlag.data.end_route_offset_millis,
+          data: { ...currOverride.data },
+          type: 'overriding',
+        };
+      }
```
Comment (jwooning) [inline src/actions/cached.js]: the events on the timeline can just overlap, If you give the `user_flag` a higher `z-index` it should be fine without splitting the other events

Comment (incognitojam) [reply src/actions/cached.js]: oh, that's a lot better, thanks!

**Pull request merged**